docs/sources/administration/correlations/correlation-configuration/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Correlation
+weight: 10
+---
+
+# Correlation
+
+Each correlation is configured with the following options:
+
+**Label**
+: Link label, shown in a visualization
+
+**Description**
+: Optional description
+
+**Source data source**
+: The source of results that have links displayed
+
+**Results field**
+: Defines where the link is shown in a visualization
+
+**Target query**
+: The target query run when a link is clicked
+
+**Transformations**
+: Optional manipulations to the source data included passed to the target query
+
+Learn how to create correlations using the [Administration page]({{< relref "./create-a-new-correlation#create-a-correlation-in-administration-page" >}}) or with [provisioning]({{< relref "./create-a-new-correlation#create-a-correlation-with-provisioning" >}}).
+
+## Source data source and result field
+
+Links are shown in Explore visualizations for the results from the correlation’s source data source. A link is assigned to one of the fields from the result provided in the correlation configuration (the results field). Each visualization displays fields with links in a different way ([Correlations in Logs Panel]({{< relref "./use-correlations-in-visualizations#correlations-in-logs-panel">}}) and see [Correlations in Table]({{< relref "./use-correlations-in-visualizations#correlations-in-table">}}))
+
+## Target query
+
+The target query is run when a link is clicked in the visualization. You can use the query editor of the selected target data source to specify the target query. Source data results can be accessed inside the target query with variables.
+
+### Correlation Variables
+
+You can use variables inside the target query to access the source data related to the query. Correlations use [Grafana variable syntax]({{< relref "/docs/grafana/latest/dashboards/variables/variable-syntax" >}}). Variables are filled with values from the source results when the link is clicked. There are two types of variables you can use:
+
+- [field variables]({{< relref "/docs/grafana/latest/panels-visualizations/configure-data-links#field-variables" >}}) (allows to access field values and labels)
+- correlation variables (allows to access field values and transformations)
+
+Example: If source results contain a field called “employee”, the value of the field can be accessed with:
+
+- A field variable ${\_\_data.fields.employee}
+- A correlation variable that maps the field value above to ${employee}
+
+Additionally to mapping field values to shorter variables names, more correlation variables can be created by applying transformations to existing fields.
```

Comment:
"In addition to mapping field values to shorter variable names, more correlation variables can be created by applying transformations to existing fields"

Reply:
Updated in 03082519f4f4657509cac9e904e6f0fc4a28f873.